theme/templates/includes/notifications-app.html

```diff
@@ -1,3 +1,4 @@
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.js"></script>
 <li id="notifications-dropdown">
```

Comment:
Provide a vue context to allow this to be independent for now, until future consolidation. Django includes are rendered independently then combined in the parent template.